admin/page/ajax-edit-produk.php

```diff
@@ -0,0 +1,264 @@
+                .catch( error => {
+                    console.error( error );
+                } );
+            
+            $("#editor4").click(function (e) { 
+                e.preventDefault();
+                $("#deskripis2").val($("#editor4").html());
+                $(this).keyup(function () { 
+                    $("#deskripsi2").val($("#editor4").html());
+                });
+            });
+        </script>
+        <?php
+    }
+    function change_file(){
+        ?>
+        <div>
+            <label for='download'>File</label>
+            <input type='file' name='download' id='download' class='form-control-file ml-auto'>
+        </div>
+        <?php
+    }
+
+
+    ?>
+
+    <div class="form-inline">
+        <label>ID</label>
+        <input type="text" name="id_produk" class="form-control ml-auto" value="<?php echo $id ?>" readonly required>
+    </div>
+    <div class="form-inline">
+        <label>Nama Produk</label>
+        <input type="text" name="nama_produk" placeholder="Nama Produk" class="form-control ml-auto" value="<?php echo $nama_produk ?>" required>
+    </div>
+    <div class="form-inline">
+        <label>Jenis Produk</label>
+        <input type="text" name="jenis_produk" class="form-control ml-auto" value="<?php echo $jenis_produk ?>" readonly required>
+    </div>
+    <?php
+
+    switch ($jenis_produk) {
+        case 'Senjata':
+            ?>
+            <div class="form-inline">
+                <label>Kategori</label>
+                <select class="form-control ml-auto" name="kategori" required>
+                    <option <?php if($kategori=="Rifles"){echo "selected";} ?> >Rifles</option>
+                    <option <?php if($kategori=="Sniper Rifles"){echo "selected";} ?> >Sniper Rifles</option>
+                    <option <?php if($kategori=="Machine Guns"){echo "selected";} ?> >Machine Guns</option>
+                    <option <?php if($kategori=="Grenade Launcher"){echo "selected";} ?> >Grenade Launcher</option>
+                    <option <?php if($kategori=="Mortir"){echo "selected";} ?> >Mortir</option>
+                    <option <?php if($kategori=="Pistols"){echo "selected";} ?> >Pistols</option>
+                    <option <?php if($kategori=="Shotguns"){echo "selected";} ?> >Shotguns</option>
+                    <option <?php if($kategori=="Submachine Guns"){echo "selected";} ?> >Submachine Guns</option>
+                </select>
+            </div>
+            <?php
+            add_spesifikasi();
+            add_deskripsi();
+            break;
+
+        case 'Munisi':
+            ?>
+            <div class="form-inline">
+                <label>Kategori</label>
+                <select class="form-control ml-auto" name="kategori" required>
+                    <option <?php if($kategori=="Small Caliber"){echo "selected";} ?> >Small Caliber</option>
+                    <option <?php if($kategori=="Big Caliber"){echo "selected";} ?> >Big Caliber</option>
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)